plugins/defaultplugins/l2plugin/vppcalls/intf/channelIntf.go

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2017 Cisco and/or its affiliates.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package intf
+
+import (
+	govppapi "git.fd.io/govpp.git/api"
+)
+
+//ChannelIntf is interface for send request to VPP channel
+type ChannelIntf interface {
```

Comment:
find some better naming for this package/interface type(something like iface.VPPChan, ifvpp.Channel etc) to make it more descriptive

Reply:
ok